src/Adapter/MSTest.TestAdapter/Extensions/ExceptionExtensions.cs

```diff
@@ -32,6 +32,10 @@ internal static Exception GetRealException(this Exception exception)
         return exception;
     }
 
+    internal static bool IsOperationCanceledExceptionFromToken(this Exception ex, CancellationToken cancellationToken)
```

Comment:
Is it intentional this doesn't handle TaskCanceledException?

Reply:
If not intentional, it may be a good idea to rename the method when fixing the impl. Something like `IsOperationCanceledOrTaskCanceledExceptionFromToken` or `IsOperationOrTaskCanceledExceptionFromToken` or anything that clearly expresses the intent.

Reply:
Ah okay. That's good then!